### PR TITLE
DOCS-2035: Add example snippets to movement sensor, power sensor, sensor

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

File: lib/src/components/power_sensor/power_sensor.dart [hunk 1]
```diff
@@ -11,25 +10,48 @@ typedef Current = GetCurrentResponse;
...
+  /// If a sensor is not configured correctly or fails to read a
+  /// piece of data, it will not appear in the readings dictionary.
+  ///
+  /// ```
+  /// var readings = await myPowerSensor.readings();
+  /// ```
   Future<Map<String, dynamic>> readings({Map<String, dynamic>? extra});
 
-  /// Get the voltage (volts) and boolean isAC
+  /// Get the voltage (in volts) and whether the power is
```
Comment (purplenicole730) [inline lib/src/components/power_sensor/power_sensor.dart:22]: nit: in other comments, the units were not in parenthesis.

File: lib/src/components/power_sensor/power_sensor.dart [hunk 2]
```diff
@@ -11,25 +10,48 @@ typedef Current = GetCurrentResponse;
...
+  ///
+  /// ```
+  /// var voltageObject = await myPowerSensor.readings();
+  /// double voltageInVolts = voltageObject['volts'];
+  /// bool isItAC = voltageObject['isAc'];
+  /// ```
   Future<Voltage> voltage({Map<String, dynamic>? extra});
 
-  /// Get the current (amperes) and boolean isAC
+  /// Get the current (in amperes) and whether the current
```
Comment (purplenicole730) [inline lib/src/components/power_sensor/power_sensor.dart:32]: same as above

File: lib/src/components/movement_sensor/movement_sensor.dart [hunk 3]
```diff
@@ -18,31 +18,72 @@ abstract class MovementSensor extends Resource {
   static const Subtype subtype = Subtype(resourceNamespaceRDK, resourceTypeComponent, 'movement_sensor');
 
   /// Obtain the measurements/data specific to this [MovementSensor]
-  /// If a sensor is not configured to have a measurement or fails to read a piece of data, it will not appear in the readings dictionary.
+  /// If a sensor is not configured to have a measurement or fails to read a piece of data,
```
Comment (purplenicole730) [inline lib/src/components/movement_sensor/movement_sensor.dart:21]: nit: comment seems long compared to others

File: lib/src/components/movement_sensor/movement_sensor.dart [hunk 4]
```diff
@@ -18,31 +18,72 @@ abstract class MovementSensor extends Resource {
...
-  /// Get the current [GeoPoint] (latitude, longitude) and altitude (mm)
+  /// Get the current [GeoPoint] (latitude, longitude) and altitude (mm).
+  ///
+  /// ```
+  /// var position = await myMovementSensor.position();
+  /// ```
   Future<Position> position({Map<String, dynamic>? extra});
 
-  /// Get the current linear velocity as a [Vector3] with x, y, and z axes represented in mm/sec
+  /// Get the current linear velocity as a [Vector3] with x, y, and z axes represented in mm/sec.
```
Comment (purplenicole730) [inline lib/src/components/movement_sensor/movement_sensor.dart:36]: nit: comment is long compared to others

**Review by purplenicole730 — APPROVED** [review]
I realized there was also effort to standardize the comments, so I added some small nits. Otherwise, LGTM!

**Review by njooma — APPROVED** [review]

**Comment by JessamyT** [comment]
@purplenicole730 Ooh I was unaware of this standardization. I personally don't see a problem with some comments being longer than others, since some methods require more explanation and some are highly self-explanatory. I'd like to leave those as they are if that's okay?

File: lib/src/components/power_sensor/power_sensor.dart (hunk 1 above)
Comment (JessamyT) [reply lib/src/components/power_sensor/power_sensor.dart:22]:
```suggestion
  /// Get the voltage in volts, and whether the power is
```

File: lib/src/components/power_sensor/power_sensor.dart (hunk 2 above)
Comment (JessamyT) [reply lib/src/components/power_sensor/power_sensor.dart:32]:
```suggestion
  /// Get the current in amperes, and whether the current
```

**Pull request merged**